src/main/java/com/ticketease/te/member/Login/LoginService.java

```diff
@@ -0,0 +1,33 @@
+package com.ticketease.te.member.Login;
+
+import com.ticketease.te.member.Member;
+import com.ticketease.te.member.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@Service
+public class LoginService implements UserDetailsService {
+
+    private final MemberRepository memberRepository;
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        Optional<Member> findMember = memberRepository.findByNickName(username);
+        if (findMember.isEmpty()){
+            throw new UsernameNotFoundException("계정정보를 확인해주세요");
+        }
```

Comment:
optional을 사용하려는 노력 아주좋습니다. 추가적으로 더 챙기면 좋을것이 있는데요! 
optional을 사용했지만 위 코드는 결국 optional을 사용하지 않던 시절의 if (findMember!= null) 코드 구문과 크게 다른 모양이 되지 않습니다.

optional을 사용할 땐 optional에서 제공하는 함수를 적극 사용해야 그 효용을 더 이끌어낼 수 있어요~
```suggestion
Member m = memberRepository.findByNickName(username)
                .orElseThrow(() -> new UsernameNotFoundException("계정정보를 확인해주세요"));
```

Reply:
네 Optional을 활용하니 코드가 더 간결해지네요!
피드백을 적용했습니다.

Reply:
https://github.com/f-lab-edu/TicketEase/pull/13/commits/63287bbb2bdaf73adbd465ba2f060046cd53a93b

피드백 적용했습니다!